content/en/logs/explorer/search_syntax.md

```diff
@@ -49,19 +49,9 @@ Use the search bar's autocomplete feature to complete your query using existing
 
 ## Escaping of special characters
 
-The following characters are considered special: `+` `-` `=` `&&` `||` `>` `<` `!` `(` `)` `{` `}` `[` `]` `^` `"` `“` `”` `~` `*` `?` `:` `\`, and `/` require escaping with the `\` character.
+The following characters are considered special: `+` `-` `=` `&&` `||` `>` `<` `!` `(` `)` `{` `}` `[` `]` `^` `"` `“` `”` `~` `*` `?` `:` `\`, and `/` require escaping with the `\` character. You can’t search for special characters in a log message. However, you can search for special characters when they’re inside of  an attribute. To search for special characters, parse them into an attribute with the [grok parser][1], and then search for logs that contain the attribute.
 
-**Note**: These characters can be escaped, but are not searchable in logs search. To search for special characters, parse them into an attribute with the [grok parser][1], and then search for logs that contain the attribute.
-
-## Attributes search
-
-### Message attribute search
-
-To search for logs that contain `user=JaneDoe` in the message attribute use the following search:
-
-```
-user\=JaneDoe
-```
+## Attributes searchå
```

Comment:
Is there a reason this header is used twice and with a special character?

Reply:
Oh, that's very weird. I don't remember adding that in there. Will remove.